src/udev/net/link-config.c

```diff
@@ -257,10 +258,14 @@ int link_config_get(LinkConfigContext *ctx, sd_device *device, LinkConfig **ret)
         if (r < 0)
                 return r;
 
-        r = rtnl_get_link_info(&ctx->rtnl, ifindex, &iftype, NULL);
+        r = rtnl_get_link_info(&ctx->rtnl, ifindex, &iftype, &flags);
         if (r < 0)
                 return r;
 
+        /* Do not configure loopback interfaces by .link files. */
+        if (flags & IFF_LOOPBACK)
+                return -ENOENT;
```

Comment:
i think the patch is OK, but I also have the suspicion there might be something we are missing here, i.e. people configuring weird shit on the loopback device (such as a low MTU) for some strange usecase.

But i guess the only way to determine if that's indeed the case, is by just making the change, and seeing if anyone complains.

commit lgtm, hence.